doc-site/vite.pages.config.ts

```diff
@@ -0,0 +1,11 @@
+import type { UserConfig } from 'vite'
+
+import react from '@vitejs/plugin-react'
+import mdx from 'vite-plugin-mdx'
+import pages from 'vite-plugin-react-pages'
+
+module.exports = {
+  jsx: 'react',
+  plugins: [react(), mdx(), pages()],
```

Comment:
Why create a new vite config for doc-site? There is already [a vite config](https://github.com/vitejs/vite-plugin-react-pages/blob/7e0356c0df3dec54fd86839dd3f11f359f1e8c34/doc-site/vite.config.ts).